### ISSUE TITLE
Update TS typings for Popper

### ISSUE THREAD

**Issue by rolandjitsu** [souporserious/react-popper#101]
I've just update MUI next to beta 37 and with it popper at `v1.14` (previously it was `v1.12`). But not the TS compiler would complain about wrong typings:
```
node_modules/react-popper/react-popper.d.ts(38,17): error TS2503: Cannot find namespace 'PopperJS'.
```

I guess instead of:
```ts
import PopperJS from "popper.js";
const v: PopperJS.Placement;
```
we should use:
```ts
import {Placement} from "popper.js";
const v: Placement;
```
or
```ts
import * as PopperJS from "popper.js";
const v: PopperJS.Placement;
```

**Comment by ablamunits** [comment]
This is breaking for us as well. I believe that this is related to the following change in `Popper.js` released just now in 1.14.0, where `Popper` is no longer a namespace.

https://github.com/FezVrasta/popper.js/commit/3315a6823f1685684fccf45f3fde407f1ca3716c#diff-01613ef3d5a6b03e140f807e6a8379bb

@FezVrasta  @souporserious any hints? :)

**Comment by FezVrasta** [comment]
Oh fuck, I hate TypeScript, it's so fucking cryptic... Would it be a solution if a dropped support for it from Popper.js at completely? I can't stand it anymore 😩

**Comment by rolandjitsu** [comment]
😆 @FezVrasta I don't think you need to. Changing the way this lib declares the import should suffice. There is already a PR here that addresses the typings, it just needs to be changed slightly.

I've just testd with `import * as PopperJS from "popper.js";` instead of `import PopperJS from "popper.js";` and it seems to work.

Ideally you'd always import the necessary exported objects from the lib instead of everything:
```ts
// e.g.
import {Placement, Modifiers} from 'popper.js';
```

**Comment by mayrbenjamin92** [comment]
The simplest fix for now is to just add "popper.js": "1.12" to your package.json (remove package-lock.json contents for one npm i). This way you will overwrite your 1.14 that gets installed as a dep of mui beta-37 with 1.12!

**Comment by FezVrasta** [comment]
Fixed in 0.8.3
